tests/unit/Entity/EntityTest.php

```diff
@@ -30,13 +30,6 @@
 abstract class EntityTest extends \PHPUnit_Framework_TestCase {
 
 	/**
-	 * Returns several more or less complex claims
-	 *
-	 * @return Claim[]
-	 */
-	public abstract function makeClaims();
```

Comment:
This seems unrelated. If this is unused (I just checked and yes, it is unused) this should be a separate patch, to be merged before this big one.